README.md

```diff
@@ -1,2 +1,45 @@
-# Hello-Microverse
-Understanding Git and GitHub flow
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello World README file</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] Great work on using the Microverse template for your `README` and customizing it to reflect your project! However, it appears that you have missed several sections from the content table.
To ensure that your `README` is complete and meets the project requirements, I suggest adding all of the required sections from the [Microverse template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md). The only section that is optional to add is the FAQ section. 👌